### PR TITLE
minor public doc changes

### PR DESCRIPTION
**Pull request by kevinlind**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by nporter-adbe — APPROVED** [review]

File: code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/Identity.java [hunk 1]
```diff
@@ -24,7 +24,9 @@
 
 import static com.adobe.marketing.mobile.edge.identity.IdentityConstants.LOG_TAG;
 
-
+/**
+ * Defines the public interface for the AEP Edge Identity extension.
```
Comment (emdobrin) [inline code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/Identity.java:28]: This may get confused with the java interface meaning. Maybe reword to..
`Defines the public APIs..`

File: code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java [hunk 2]
```diff
@@ -21,7 +21,10 @@
 import static com.adobe.marketing.mobile.edge.identity.IdentityConstants.LOG_TAG;
 
 /**
- * Represents an identity item
+ * Identity is used to clearly distinguish people that are interacting with digital experiences.
+ * An {@code IdentityItem} is intended to be included in an instancd of {@link IdentityMap}.
```
Comment (emdobrin) [inline code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java:25]: typo: an instance

File: code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java [hunk 3]
```diff
@@ -47,37 +50,41 @@ public IdentityItem(final String id, final AuthenticatedState authenticatedState
...
 	/**
+	 * The state this identity is authenticated.
 	 * @return Current {@link AuthenticatedState} for this item
 	 */
 	public AuthenticatedState getAuthenticatedState() {
 		return authenticatedState;
 	}
 
 	/**
+	 * Indicates this identity is the preferred identity.
```
Comment (emdobrin) [inline code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java:86]: Can we reword to : `Indicates if this identity is the preferred identity or not.` ?
 
Also, should we give a hint on this that there can only be one active primary identity at any point in time?

Comment (kevinlind) [reply code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java:86]: I'll change to `Indicates if this identity is the preferred identity`. 
I would prefer to link to external docs on how to use identifiers and the IdentityMap.  I don't think this API is the right place to detail how an XDM schema should be created and used. However, a quick search of the Identity Service documentation doesn't give any description on how to use the `primary` field. https://experienceleague.adobe.com/docs/experience-platform/identity/home.html?lang=en

Comment (emdobrin) [reply code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java]: One thing I am concerned with that if the link changes our API docs will be incorrect (unless the sdk is updated and the customer does an sdk update too), so I would prefer to still keep this in our docs where we do updates more often.

**Review by emdobrin — APPROVED** [review]
